aws/table_aws_dlm_lifecycle_policy.go

```diff
@@ -24,7 +24,7 @@ func tableAwsDLMLifecyclePolicy(_ context.Context) *plugin.Table {
 			KeyColumns: plugin.SingleColumn("policy_id"),
 			Hydrate:    getDLMLifecyclePolicy,
 			IgnoreConfig: &plugin.IgnoreConfig{
-				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFound"}),
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFound", "ResourceNotFoundException"}),
```

Comment:
@ParthaI do we need to add both the error codes in the ignore error func?

Reply:
@misraved, The API version which was used earlier (`v1.14.2`) that returns `ResourceNotFound`, but as per the current version (`v1.15.6`) the API is returning the `ResourceNotFoundException` if resource is not available.

 Removed the `ResourceNotFound` from ignore config.